examples/case-3/case-3.js

```diff
@@ -0,0 +1,483 @@
+                              if (props.handleDrop)
+                                props.handleDrop(monitor.getItem(), dropTarget);
+                            }
+                          },
+                          // collecting function
+                          function(connect, monitor) {
+                            return {
+                              connectDropTarget: connect.dropTarget(),
+                              isOver: monitor.isOver(),
+                              canDrop: monitor.canDrop()
+                            };
+                          }
+                        )(_DropOrganismGlowView);
+
+/**
+ * ParentDrakeColumn - shows mother/father image and genome
+ */
+class ParentDrakeColumn extends React.Component {
+
+  static propTypes = {
+    id: React.PropTypes.string.isRequired,
+    sex: React.PropTypes.string.isRequired,
+    drake: React.PropTypes.object.isRequired,
+    isDrakeEditable: React.PropTypes.bool.isRequired,
+    hiddenAlleles: React.PropTypes.arrayOf(React.PropTypes.string),
+    alleleChanged: React.PropTypes.func.isRequired
+  }
+
+  render = () => {
+    const { id, sex, drake, isDrakeEditable, hiddenAlleles, alleleChanged } = this.props,
+          drakeLabelID = `${sex}-drake-label`,
+          columnLabel = `${sex.charAt(0).toUpperCase() + sex.slice(1)} Drake`,
+          drakeOrgID = `${sex}-drake`;
+    return (
+      <div id={id} className="column">
+
+        {/* parent drake label */}
+        <div id={drakeLabelID} className="column-label">{columnLabel}</div>
+
+        {/* parent drake image */}
+        <GeniBlocks.OrganismGlowView
+          id={drakeOrgID} className='parent-drake' org={drake} color='#FFFFAA' size={200} />
+
+        {/* parent drake genome */}
+        <GeniBlocks.GenomeView
+          className='parent-genome' org={drake} hiddenAlleles={hiddenAlleles}
+          editable={isDrakeEditable} alleleChanged={alleleChanged} />
+      </div>
+    );
+  }
+}
+
+/**
+ * Center panel has target drakes, breedng pen, breed button, etc.
+ */
+class _Case3Center extends React.Component {
+
+  static propTypes = {
+    targetDrakes: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
+    targetsMatched: React.PropTypes.object.isRequired,
+    targetDrakeSize: React.PropTypes.number.isRequired,
+    glowColor: React.PropTypes.string.isRequired,
+    matchedColor: React.PropTypes.string.isRequired,
+    clutch: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
+    clutchSize: React.PropTypes.number.isRequired,
+    handleBreed: React.PropTypes.func.isRequired,
+    requiredMoveCount: React.PropTypes.number.isRequired,
+    moveCount: React.PropTypes.number.isRequired,
+    handleDrop: React.PropTypes.func.isRequired
+  }
+
+  state = {
+    selectedIndex: null
+  }
+
+  constructor(props) {
+    super(props);
+    document.addEventListener('keydown', () => enableButtonFocusHighlight());
+  }
+
+  breed = () => {
```

Comment:
Not sure, that's why I was asking. :wink: But in general an arrow function `() => { ... }` is equal to `function () { ... }.bind(this)`. But I've never seen it in the class declaration. So, I was wondering if that works in this case and e.g. you can just call `setTimeout(this.breed, 1)` instead of `setTimeout(this.breed.bind(this), 1)`.